### PR TITLE
Add BitXor, BitXorAssign, and symmetric difference

### PR DESCRIPTION
**Pull request by nwn**
Adds a symmetric_difference operation like in the `std::collections` sets, as well as the analogous BitXor operations.

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -564,6 +600,33 @@ impl <'a> BitOrAssign for FixedBitSet
...
+    fn bitxor(self, other: &FixedBitSet) -> FixedBitSet {
+        let (short, long) = {
+            if self.len() <= other.len() {
+                (&self.data, &other.data)
+            } else {
+                (&other.data, &self.data)
+            }
+        };
+        let mut data = long.clone();
+        for (i, &block) in short.iter().enumerate() {
```
Comment (jrraymond) [inline src/lib.rs]: Can you use `zip` here? It is more idiomatic, safer, and potentially more performant.

Comment (nwn) [reply src/lib.rs]: Good idea. The latest commit changes BitAnd, BitOr, and BitXor to use `zip` here.

**Comment by jrraymond** [comment]
@bluss yes

**Comment by bluss** [comment]
@jrraymond do you have a user on crates.io? Username? :)

**Comment by bluss** [comment]
Do you think it's ok to use cargo release to do release packages + tags? It works smoothly with little overhead that way. Also, it is of course time to reevaluate the minimum rust version. Either very conservatively while staying on 0.1 or bumping to 0.2 and bumping the minimum version a bit more. Something that goes well with the next version of petgraph would be good.

**Comment by jrraymond** [comment]
The username should be jrraymond.

**Comment by jrraymond** [comment]
Lets open a seperate issue to discuss bumping the minor version? It would be easier for other users to find a comment on.

I think the current release protocol is good.

**Comment by bluss** [comment]
Owner invite sent. Don't ever feel like it's a burden, just push it back on me if you don't want to do it. That said, every crate needs multiple owners. Thanks for coming aboard. :slightly_smiling_face:

**Comment by jrraymond** [comment]
accepted. :)

**Pull request merged**